### ISSUE TITLE
handle command line arguments

### ISSUE THREAD

**Issue by zom-ponks** [zom-ponks/rpcs3-gameupdater#4]
Manage command line arguments, at least we should have

* configuration file override
* path setup (RPCS3 location, download path)
* a "dry run" switch (just display potential actions taken)
* help switch

**Comment by zom-ponks** [comment]
Basic functionality in commit [d906534b1149a9d8e330ec0187ec4f7013f3cc88](https://github.com/zom-ponks/rpcs3-gameupdater/commit/d906534b1149a9d8e330ec0187ec4f7013f3cc88)

Still needs a bit of work

**Comment by John-Gee** [comment]
I've added a few comments, but that looks good.

**Comment by zom-ponks** [comment]
Yeah, I realized too late that I should've done a PR instead, apologies for that!

**Comment by John-Gee** [comment]
It's all good.
